dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/internal/net/TracingRequestInterceptorTest.kt

```diff
@@ -189,7 +192,7 @@ internal class TracingRequestInterceptorTest {
         assertThat(transformedRequest.method()).isEqualTo(fakeMethod)
         assertThat(transformedRequest.header(key)).isEqualTo(value)
         inOrder(mockSpan) {
-            verify(mockSpan).setTag("http.url", fakeUrl)
+            verify(mockSpan).setTag("http.url", fakeWhitelistedDomainUrl.toLowerCase())
```

Comment:
Why the "toLowerCase()"? Urls are case sensitive for everything after the domain name, changing the case means it's not the same URL. You should instead generate a `fakeWhitelistedDomainUrl` based on the lowercase of `fakeWhitelistedDomain`

Reply:
the fake urls are only "http://domain/" now. The `request.url().host()` is actually transforming the host to lowercase so I chose the easy way to assert those as will not affect any test...but yeah it is a bit ugly. Need to rework also those url generators.

Reply:
Because we also generate domains with capital letters but OkHttp transforms the domain name to lowercase when you call `request.url()`.  Maybe I will create a method that computes the *expectedUrl* based on the *fakeUrl* by lowering the case of the *domain* in it.